docs/manual/index-checker.tex

```diff
@@ -816,8 +816,8 @@
 
   lst.add("hello");    // OK
   lstGO.add("hello");  // OK
-\end{Verbatim}
 
```

Comment:
It is not desirable to move this blank link within the `Verbatim` environment.

Reply:
Resolved